docker-compose/bootstrapper-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+# Ensure that expected dirs exist.
+mkdir -p "${CONCORDIUM_NODE_CONFIG_DIR}" "${CONCORDIUM_NODE_DATA_DIR}"
```

Comment:
Would it not be better to create this directory directly in the build and avoid this needless indirection via the start script, just setting the entrypoint to the bootstrapper.

Reply:
good idea, done